packages/ui/components/input-stepper/src/LionInputStepper.js

```diff
@@ -66,6 +71,8 @@ export class LionInputStepper extends LocalizeMixin(LionInput) {
     this.formatter = formatNumber;
     this.min = Infinity;
     this.max = Infinity;
+    /** @type {string | undefined} */
```

Comment:
Maybe some description, so that this will be picked up in our api tables?
https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaValueText